libminifi/src/FlowFileRecord.cpp

```diff
@@ -50,17 +50,17 @@ std::shared_ptr<FlowFileRecord> FlowFileRecord::DeSerialize(const std::string& k
   std::string value;
 
   if (!flowRepository->Get(key, value)) {
-    logger_->log_error("NiFi FlowFile Store event %s can not found", key);
+    logger_->log_error("NiFi FlowFile Store event {} can not found", key);
     return nullptr;
   }
   io::BufferStream stream(value);
 
   auto record = DeSerialize(stream, content_repo, container);
 
   if (record) {
-    logger_->log_debug("NiFi FlowFile retrieve uuid %s size " "%" PRIu64 " connection %s success", record->getUUIDStr(), stream.size(), container.to_string());
+    logger_->log_debug("NiFi FlowFile retrieve uuid {} size {} connection {} success", record->getUUIDStr(), stream.size(), container.to_string());
   } else {
-    logger_->log_debug("Couldn't deserialize FlowFile %s from the stream of size " "%" PRIu64, key, stream.size());
+    logger_->log_debug("Couldn't deserialize FlowFile {} from the stream of size " "{}", key, stream.size());
```

Comment:
There are many instances of having the string literal split into two parts, where `PRIu64` or similar format specifiers were used. Splitting is no longer necessary with fmt, so I think we should join those strings back to a single string literal.
```suggestion
    logger_->log_debug("Couldn't deserialize FlowFile {} from the stream of size {}", key, stream.size());
```

Reply:
Nice catch, I've missed these. fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1670/commits/e4f21d88ae2f0df7bd9f47a800fd2779f38a91a6